Variable files with either `.yml` or `.yaml` are not being rendered

The error `[ERROR] levant/command: 'job' stanza not found` is raised every time as template is rendered using a `.yaml/.yml` variable file extension.

--- a/levant/templater.go
+++ b/levant/templater.go
@@ -17,9 +17,9 @@ import (
 )
 
 const (
-	terraformVarExtention = "tf"
-	yamlVarExtension      = "yaml"
-	ymlVarExtension       = "yml"
+	terraformVarExtention = ".tf"
+	yamlVarExtension      = ".yaml"
+	ymlVarExtension       = ".yml"
 )
 
 // RenderTemplate is the main entry point to render the template based on the
@@ -118,8 +118,7 @@ func renderYAMLVarsTemplate(src, variableFile string, flagVars *map[string]strin
 
 	// Setup the template file for rendering
 	t := template.New("jobTemplate")
-	tempF, _ := ioutil.ReadFile(src)
-	if t, err = t.Parse(string(tempF)); err != nil {
+	if t, err = t.Parse(string(src)); err != nil {
 		return
 	}
 

